src/rgw/rgw_main.cc

```diff
@@ -303,6 +303,7 @@ int radosgw_Main(int argc, const char **argv)
   ceph::crypto::init_openssl_engine_once();
 
   common_init_finish(g_ceph_context);
+  jaeger_tracing::init_jaeger();
```

Comment:
+1

Reply:
@yuval can you explain what needs to be changed

Reply:
@suab321321 why not use `jaegertracing::init_tracer`